packages/contracts/envParsed.ts

```diff
@@ -15,9 +15,9 @@ const env = {
 const envSchema = z
   .object({
     NETWORK_TESTNET: z.string().transform((x) => x as Address),
-    NETWORK_MAINNET: z.string().transform((x) => x as Address),
+    NETWORK_MAINNET: z.string().transform((x) => x as Address).optional(),
```

Comment:
```suggestion
    NETWORK_MAINNET: z.string().transform((x) => x as Address).optional(),
```
this should be required, also we can add a default value pointing to OPTIMISM or BASE. Same for testnet env variables.